example/main.go

```diff
@@ -35,10 +41,15 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 	"sigs.k8s.io/controller-runtime/pkg/source"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/types"
 )
 
 var (
-	log = logf.Log.WithName("example-controller")
+	log    = logf.Log.WithName("example-controller")
+	scheme = runtime.NewScheme()
```

Comment:
we default to the kubernetes scheme elsewhere -- it's probably safe to do so here as well.